clock-bound-d/Cargo.toml

```diff
@@ -20,11 +20,12 @@ clap = "2.33"
 tokio = { version = "1.12", features = ["sync"] }
 chrono = "0.4.19"
 byteorder = "1.4.3"
+uds = "0.2.6"
 
 [[bin]]
 name = "clockboundd"
 path = "src/main.rs"
 doc = false
 
 [badges]
-github = { repository = "aws/clock-bound-d"}
+github = { repository = "aws/clock-bound"}
```

Comment:
Why are we changing this to `aws/clock-bound`? We have the badge set to `-c` and `-d` based on the client and daemon

Reply:
I do recall that there was some issue here; is this meant to fix it? In any case please rebase unrelated changes like this into a separate commit.

Reply:
The issue is if you put it in the browser, it is 404. Ideally, those are 2 repos.

Reply:
Haven't tested this, but I'm pretty sure we can remove this entirely. When writing this I attempted to add badges like this, but instead tied them into the readme generation. They are now generated via https://github.com/aws/clock-bound/blob/main/clock-bound-d/README.tpl#L1 and this line I believe doesn't do anything anymore. See clock-bound-c, this line doesn't exist: https://github.com/aws/clock-bound/blob/main/clock-bound-c/Cargo.toml

The repository info on crates.io gets pulled from https://github.com/aws/clock-bound/blob/main/clock-bound-d/Cargo.toml#L9.